2-mandatory/3-financial-times.js

```diff
@@ -15,6 +22,15 @@ function potentialHeadlines(allArticleTitles) {
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+    let min=100;
```

Comment:
DId you know, you can use `Infinity` as a starting value for situations like this where you want to find the minimum?
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity